runtime/tests/vm/vmstubs.c

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1991, 2016 IBM Corp. and others
+ * Copyright (c) 1991, 2019 IBM Corp. and others
```

Comment:
What is the purpose of this file?

Reply:
This is used when the resolvesupport.cpp is compiled outside the VM for some of our tests.  It stubs out functions that aren't actually called in the tests

Reply:
Those tests should probably be refactored/removed so they can run with the real VM, but I don't expect you to do that here.